apps/session-manager/src/services/lollipop.ts

```diff
@@ -83,3 +86,58 @@ export const generateLCParams: (
         }
       }),
     );
+
+export const deleteAssertionRefAssociation: (
+  fiscalCode: FiscalCode,
+  assertionRefToRevoke: AssertionRef,
+  eventName: string,
+  eventMessage: string,
+) => RTE.ReaderTaskEither<
+  LollipopRevokeRepo.RevokeAssertionRefDeps &
+    RedisRepositoryDeps & {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      appInsightsTelemetryClient?: any;
+    },
+  Error,
+  boolean
+> = (fiscalCode, assertionRefToRevoke, eventName, eventMessage) => (deps) => {
+  // Sending a revoke message for assertionRef
+  // This operation is fire and forget
+  pipe(
+    LollipopRevokeRepo.revokePreviousAssertionRef(assertionRefToRevoke)(deps),
+    TE.mapLeft((err) => {
+      deps.appInsightsTelemetryClient?.trackEvent({
+        name: eventName,
```

Comment:
```suggestion
        name: eventName + ".revoke-assertion-ref",
```

Reply:
Why this change, will create a new event. Now in production there are 2 different points where the event is sent as `lollipop.error.acs`